Corona-Warn-App/src/main/res/layout/home_card_container_layout.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/card_container"
+    style="@style/cardNoPadding"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="@dimen/spacing_small"
+    android:layout_marginEnd="@dimen/spacing_small"
+    android:layout_marginTop="@dimen/spacing_tiny"
+    android:layout_marginBottom="@dimen/spacing_tiny"
+    android:foreground="?selectableItemBackground"
+    android:focusable="true" />
```

Comment:
This change replaces the `FrameLayout` with a `CardLayout`.

```suggestion
<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:id="@+id/card_container"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:background="@color/colorSurface1"
    android:layout_marginStart="@dimen/spacing_small"
    android:layout_marginEnd="@dimen/spacing_small"
    android:layout_marginTop="@dimen/spacing_tiny"
    android:layout_marginBottom="@dimen/spacing_tiny"
    android:foreground="?selectableItemBackground"
    android:focusable="true"
    app:cardElevation="@dimen/elevation_strong"
    app:cardCornerRadius="@dimen/radius_card" />
```

Here's a screenshot:

![cwahome](https://user-images.githubusercontent.com/16943720/102550542-47f2f200-40be-11eb-8b50-102f7e8a9e0f.png)

Notice how the green card has rounded corners again.